connect.js

```diff
@@ -86,10 +94,11 @@ connect.prototype.close = function () {
   window.removeEventListener("message", this.eventHandler, false);
   connect.prototype.utils.closeWidget();
   this.onClose();
+  this.onEvent('CLOSED', connect.prototype.utils.metadata({}));
```

Comment:
closed event would now receive the `pageName` where the widget was closed, this would be sent form the widget

Reply:
Currently not receiving a pageName in event.data